### ISSUE TITLE
Date picker open twice, hence need to be confirmed twice on Android.

### ISSUE THREAD

**Issue by tz925** [react-native-community/react-native-datetimepicker#122]
# Bug

 ```const onChange = (event, selectedDate) => {
    const currentDate = selectedDate || date;

    setDate(currentDate);
    setShow(Platform.OS === 'ios' ? true : false);
  };
```
this piece of code show as an example on homepage readme file will produce the date picker, but on android, the picker will be opened twice, hence need to be confirmed twice on Android. 

# fix

move ``` setShow(Platform.OS === 'ios' ? true : false);``` to second line, right after currentDate declaration fixes issue on my end. Also ```? true : false``` is redundant. 

# suggestion

modify the example code on readme file.

# ReactNative version 0.61, expo version 36.0.0, datetimepicker version 2.1.0

**Comment by VictorhMalheiro** [comment]
This helped me a lot, simple and efficient solution was racking my brains! Thanks!

**Comment by luancurti** [comment]
@tz925 would you like to open a PR to fix this?

**Comment by tz925** [comment]
> @tz925 would you like to open a PR to fix this?

Sure I will do a PR.

**Comment by tz925** [comment]
Please check to see if my PR is submitted correctly... First time doing this 😂

**Comment by luancurti** [comment]
Fixed by #124

**Comment by tsleolima** [comment]
thanks man :smile: :heart:
